repository-data/development/src/main/resources/hcm-config/hst/hosts.yaml

```diff
@@ -41,6 +42,7 @@ definitions:
               hst:mountpoint: /hst:hst/hst:sites/common
               jcr:primaryType: hst:mount
             jcr:primaryType: hst:virtualhost
+          hst:scheme: http
```

Comment:
Is it worth adding a comment: # No SSL in local dev environment for simplicity

Reply:
Can't comment in YAML, see above